.appveyor.yml

```diff
@@ -20,7 +20,7 @@ install:
         [System.IO.Compression.ZipFile]::ExtractToDirectory("C:\sbt-bin.zip", "C:\sbt")
       }
   - SET PATH=C:\sbt\sbt\bin;%PATH%
-  - SET SBT_OPTS=-XX:MaxPermSize=2g -Xmx4g -Dfile.encoding=UTF8
+  - SET SBT_OPTS=-XX:MaxPermSize=2g -Xmx4g -Dfile.encoding=UTF8 -Dsbt.supershell=never
```

Comment:
We should split this commit out to another PR.
Maybe we should turn it off by default in CI.
https://github.com/sbt/sbt/blob/ca5f978df495a5a975302cedba73986d2abaa262/main/src/main/scala/sbt/Defaults.scala#L332

Reply:
Ok. I added #4590.  I'll remove this commit from this PR.